reservation/test_schedule_model.py

```diff
@@ -0,0 +1,64 @@
+from reservation.models import Schedule, Appointment
+from datetime import datetime
+import pytest
+
+
+START_DAY = datetime(2023, 4, 17, 10, 0, 0)
+END_DAY = datetime(2023, 4, 17, 18, 0, 0)
+MEETING_TIME = 60
+
+
+@pytest.fixture
+def persisted_schedule(schedule):
+    schedule.professional_id.save()
+    schedule.save()
+    return schedule
+
+
+@pytest.fixture
+def persisted_schedule_pool(appointment, typeOfJob, persisted_schedule, professional, client2):
```

Comment:
same as above,
Please consider this https://www.inspiredpython.com/article/five-advanced-pytest-fixture-patterns

One method with two parameters

Reply:
Can you explain to me the problem with what I did? And besides, I didn't quite understand the solution you offer

Reply:
see above reply

Reply:
I understand what you said but the changes you suggested should be in conftest.py? Won't this cause tests to fail? Because everyone will have to change their tests after my changes

Reply:
see above reply

Reply:
The comment here, is related to the way you create a pool not the users

Reply:
Done